examples/create-token-bridge-custom-fee-token/.env.example

```diff
@@ -0,0 +1,13 @@
+# Address of the Rollup contract
+ROLLUP_ADDRESS=
```

Comment:
I created .env.example files in the new examples to have users us a .env file. The old create-rollup examples use process.env variables but didn't load .env files. Do we want to change that?